resource-aggregate/pb/events.proto

```diff
@@ -26,8 +26,9 @@ message ResourceLinksPublished {
 message ResourceLinksUnpublished {
     string device_id = 1;
     repeated string hrefs = 2;
-    AuditContext audit_context = 3;
-    EventMetadata event_metadata = 4;
+    repeated int64 instance_ids = 3;
+    AuditContext audit_context = 4;
+    EventMetadata event_metadata = 5;
```

Comment:
breaking change - we cannot do this because we already released version

Reply:
fixed